examples/story-book/README.md

```diff
@@ -0,0 +1,31 @@
+# Story Book
+
+Story Book is a GPTScript that can generate a story based on a prompt and the number of pages you want the story to be in. It is generated in HTML format and can then be viewed
+by `index.html` which has some JS/CSS to make the story styling consistent and readable.
+
+## How to Use
+1. Install the `image-generation` tool.
+
+```shell
+git clone https://github.com/gptscript-ai/image-generation
+```
+
+2. Bootstrap the `image-generation` tool.
+
+```shell
+(
+    cd image-generation
+    make bootstrap
+)
+```
+
+3. Source the `venv` for `image-generation`.
+```
+source image-generation/venv/bin/activate
```

Comment:
Looks like this should be `venv/bin/activate` since you are already in `image-generation` folder?

Reply:
That's a good point. The command above this has `()` around it which means that it won't actually change the user's current shell. However, I can just include the source command in that script so there is no separate command.

Reply:
oh, then actually nvm.